### PR TITLE
Explore functionality

### PR DESCRIPTION
**Pull request by Malthusis**
This implements the basic Explore Deck, Resources, and some other minor improvements!
Changelog:
- Add some support for resources, currently based out of `player.ts`.
- Add `deckstore.ts`. This holds the deck that you can pull exploration events from for Energy, which gives you basic resources, or rarely Soul.
- Draw up some sprites and for the Explore deck, and basic animations.
- Remove background box for the smallest supported screen type (1280x720).

Things to do in future PRs:
- Add Resource Display.
- Make the progression upgrades cost resources.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1"> 
     <title>Kitsune</title>
```
Comment (Malthusis) [inline index.html:5]: This is required to make `@media` css queries work, much to my chagrin after a few hours of messing around...

File: src/assets/styles.css [hunk 2]
```diff
@@ -197,14 +197,8 @@ button:hover {
     height: 165px;
 }
 
-
-
-/* youre gonna want to have like, 3 boxes on top row 2 on bottom, and then absolutely position a bordered box with a background on top of them */
```
Comment (cm97878) [inline src/assets/styles.css:202]: wonder wtf i was talking about here. good notes, past me!

File: src/components/ExplorePanel.vue [hunk 3]
```diff
@@ -1,75 +1,149 @@
 <template>
-    <div v-show="active === Tab.EXPLORE && !isEventActive">
+    <div v-show="active === Tab.EXPLORE">
         <div v-if="combatStore.activeCombat" class="in-combat">
             Currently in combat, cannot explore safely.
         </div>
```
Comment (cm97878) [inline src/components/ExplorePanel.vue:5]: This can probably be removed, 'cause this only shows at home now and there can't be combat.

**Comment by cm97878** [comment]
Looks all good, clean and understandable framework for it

**Pull request merged**